deploy/terraform/mod/rbac/main.tf

```diff
@@ -112,3 +116,16 @@ resource "kubernetes_cluster_role_binding" "mayastor" {
     name      = "mayastor-cluster-role"
   }
 }
+
+## When testing sometimes things get seriously broken and we can't easily use terraform destroy
```

Comment:
I dont understand what this brings. When we destroy everything gets 'removed' including the rbac roles? Why would adding a kubectl make things any better?

Reply:
I'm not exactly sure how this happens, terraform gets into a bad state (probably from testing no doubt) but sometimes we can't use terraform destroy, and those CRD's are left and when applying again, it fails complaining they already exist.